src/Sentry/Internal/Http/CachingTransport.cs

```diff
@@ -67,10 +67,11 @@ private async Task CachedTransportBackgroundTask()
                 {
                     foreach (var filePath in Directory.EnumerateFiles(_processingDirectoryPath))
                     {
-                        File.Move(
-                            filePath,
-                            Path.Combine(_isolatedCacheDirectoryPath, Path.GetFileName(filePath))
-                        );
+                        var destinationPath = Path.Combine(_isolatedCacheDirectoryPath, Path.GetFileName(filePath));
+                        _options.DiagnosticLogger?.LogDebug("Moving unprocessed file back to cache: {0} to {1}.",
+                            filePath, destinationPath);
```

Comment:
nit-pick but I believe the below one will be slightly more clear to the end-user.
```suggestion
                        _options.DiagnosticLogger?.LogDebug("Moving unprocessed file back from cache {0} to {1}.",
                            filePath, destinationPath);

```

Reply:
You suggestion adds another call to the logger? 
Code here is moving for "processing" back to the "cache" folder. So I think the original message makes more sense